packages/api/src/validations/validations.ts

```diff
@@ -143,7 +143,9 @@ interface PresenceValidatorOptions extends WithOptionalMessage {
   allowEmptyString?: boolean
 }
 
-interface UniquenessValidatorOptions extends WithRequiredMessage {}
+interface UniquenessValidatorOptions extends WithOptionalMessage {
+  prismaClient?: PrismaClient
```

Comment:
I feel we should keep this generic and just call the property `db`. I don't think we explicitly refer to PrismaClient anywhere in an app really, access is always via `db`. Who knows, in the future you may have a client besides Prisma and this should be able to accept that as well!